masterlist.yaml

```
@@ -10254,6 +10254,34 @@ plugins:
         util: '[TES4Edit v4.0.4b](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 2
 
+  - name: 'Umbra the (Unfortunate|Connoisseur)\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/46173/'
+        name: 'Umbra the Unfortunate'
+  - name: 'Umbra the Unfortunate.esp'
+    req: [ *OBSE ]
+    msg: [ *alsoUseSilentVoice ]
+    tag:
+      - -Actors.AIPackagesForceAdd
+      - Actors.AIPackages
```

These should be sorted alphabetically, ignoring the hyphen:

```suggestion
      - Actors.AIPackages
      - -Actors.AIPackagesForceAdd
```